client/my-sites/email/paths.js

```diff
@@ -133,8 +133,13 @@ export function emailManagementForwarding( siteName, domainName, relativeTo = nu
  * @param {string} relativeTo - optional path prefix
  * @returns {string} the corresponding url
  */
-export function emailManagementPurchaseNewEmailAccount( siteName, domainName, relativeTo = null ) {
-	return emailManagementEdit( siteName, domainName, 'purchase', relativeTo );
+export function emailManagementPurchaseNewEmailAccount(
+	siteName,
+	domainName,
+	relativeTo = null,
+	urlParameters = null
```

Comment:
It might be better to use a `context`  or `sourceContext`parameter directly. The intent of the parameter is easier to discern that way.

```js
context : string = null
```